### PR TITLE
v1.14.1 geoips - VERSION RELEASE - CI sync

### PR DESCRIPTION
**Pull request by mindyls**
# Related Issues
required to close NRLMMD-GEOIPS/geoips#740
required to close 

# Reviewer Instructions
* Please confirm this PR is set up to merge from v1.14.1-release branch into main branch
* Review "Files changed" tab
* Confirm appropriate testing was completed for these updates (you may perform tests yourself, or review output included from another reviewer/assignee)

# Summary
Changes for version 1.14.1, uploaded with
1.14.1 update on repo geoips.
See release note updates in 'Files changed' tab

See NRLMMD-GEOIPS/geoips#740 for additional repositories included in this update.
See  for dev updates included in this update.

# Testing Instructions
See NRLMMD-GEOIPS/geoips#740 for testing instructions.

# Output
See NRLMMD-GEOIPS/geoips#740 for test output.

# Post Merge Steps
Once this PR has been merged, v1.14.1
will be tagged/released on the geoips main branch.

### CODE REVIEW

File: geoips/commandline/geoips_get.py
Comment (mindyls) [inline geoips/commandline/geoips_get.py:18]: Should this be removed?  Is this geoips_describe now?

File: geoips/commandline/log_setup.py
Comment (mindyls) [inline geoips/commandline/log_setup.py:211]: I think this is just 2 ways of doing the same th9ing - 3either global logging_setup_done variable, or checking if log.handlers is defined.

**Comment by mindyls** [comment]
Note RST release note failure is due to header updates to existing RST release notes.  This update will not happen again, so we should merge with that CI failing:

image

File: geoips/commandline/geoips_get.py
Comment (jsolbrig) [reply geoips/commandline/geoips_get.py:18]: I'm not sure. @evrose54?

File: geoips/commandline/log_setup.py
Comment (jsolbrig) [reply geoips/commandline/log_setup.py:211]: @evrose54?

File: geoips/commandline/geoips_get.py
Comment (evrose54) [reply geoips/commandline/geoips_get.py:18]: This is out of sync. Main has GeoIPS describe. Unless there were any new CLI updates included in this PR, which to my recollection there weren't, I'd remove this.

File: geoips/commandline/log_setup.py
Comment (evrose54) [reply geoips/commandline/log_setup.py:211]: While I implemented the global setup you specified, I can't guarantee that these are the same things. If you're comfortable making this change and can ensure that the logs aren't duplicated, I'm fine with this! Need to do more research on this before I'm positive.

Comment (mindyls) [reply geoips/commandline/log_setup.py:211]: @AJThorpe made this change at the same time you did to fix the duplicate errors :-) We've been using this version of the update, so I think they're both fine, the merge just grabbed this version.  I'm gonna just go with it.

File: geoips/commandline/geoips_get.py
Comment (mindyls) [reply geoips/commandline/geoips_get.py:18]: Just remove the entire geoips_get.py file, right?

Comment (mindyls) [reply geoips/commandline/geoips_get.py:18]: I am removing it!!

**Comment by mindyls** [comment]
Ready  to merge once we have 2 approvals and all conversations are resolved.

The one failing action is release note check - these just have header updates, and we should never have modifications to RST release note files again since we now auto-generate RST release notes.  We should merge with this one remaining failing check since these are expected changes to existing RST release notes.

Additionally, CI is now working on other repositories (geoips_clavrx, geoips_plugin_example, geoips_tropics, ryglickicane, etc), though still fails for the fortran-based repos (including recenter_tc since it requires akima).

image

image

**Review by jsolbrig — APPROVED** [review]

**Review by christopher-selman — APPROVED** [review]

File: pyproject.toml
Comment (cpcamacho) [inline pyproject.toml:63]: Does this need to be <1.7?

**Review by cpcamacho — APPROVED** [review]

**Review by evrose54 — APPROVED** [review]

File: pyproject.toml
Comment (jsolbrig) [reply pyproject.toml]: I updated this to be <1.7.0

**Review by jsolbrig — APPROVED** [review]
This is passing for me with one exception. 

`ami.tc.WV.geotiff.sh` is failing. I'm not sure if this is a difference between the data that NRL and CIRA are using or if it is something else. The data range from 0 to 255. There are three pixels that differ by -2, 5, and 1, respectively.

I do not think that this difference is enough to warrant delaying this release any longer.

**Review by cpcamacho — APPROVED** [review]

**Pull request merged**
